Jenkinsfile

```diff
@@ -65,6 +65,13 @@ node('docker && imgtec') {  // Only run on internal slaves as build takes a lot
             // Default config
             sh "cp ${params.CONFIG_FILE?.trim()} .config"
 
+            // Add development config
+            echo 'Enabling development config'
+            sh 'echo \'' \
+             + 'CONFIG_DEVEL=y\n' \
+             + 'CONFIG_LOCALMIRROR=\"https://downloads.creatordev.io/pistachio/marduk/dl\"\n' \
```

Comment:
If somebody checking out the code manually, and using the creator-platform-default-cascoda.config to build, then he won't get this local mirror option enabled? Is there any reason for keeping the local mirror option only for CI?